src/pages/guides/using/creating_rest_apis.md

```diff
@@ -153,6 +153,145 @@ curl -i -X OPTIONS https://adobeioruntime.net/...
 
 ## Securing the API endpoints
 
+### Oauth (using the Adobe Identity Management System)
+
+You can configure an action to require IMS validation for incoming requests using the following command: 
```

Comment:
It seems other parts of the document is written in 3rd person rather than 2nd person. So, for consistency this should be something like `An action can be configured ...` rather than `You can configure ...`.